src/benchmarklib/sqlite_add_indices.hpp

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include <string>
+
+namespace opossum {
+
+class BenchmarkRunner;
+
+/**
+ * Add indexes to SQLite. This is a hack until we support CREATE INDEX ourselves and pass that on to SQLite.
```

Comment:
pls revise comment, maybe doc params briefly, one might not know what a "create_indices_file" is.

 "Without this, SQLite would never finish" - never finish doing what?
"This is a hack until we support CREATE INDEX ourselves and pass that on to SQLite" - even if we'd support it, it's not a given that we'd use it, as Hyrise functions just as well without it.